packages/react-app/src/constants/discordInfo.ts

```diff
@@ -0,0 +1,9 @@
+const discordInfo = {
+  baseUrl: 'https://discord.com/channels',
+  server: '851552281249972254',
+  channel: '869964294865977425',
```

Comment:
I think we'll need to store this in .env variable to support both discord garage and bankless dao discord. Not a blocker for the PR though!

Reply:
Makes sense, we can move to that as long as it uses the `NEXT_PUBLIC` prefix so it can added to the front end site generator:

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser